apps/prairielearn/python/prairielearn/html_utils.py

```diff
@@ -327,6 +350,11 @@ def get_color_attrib(
 
 
 def inner_html(element: lxml.html.HtmlElement) -> str:
+    """Get the inner HTML of an lxml element.
+
+    Returns:
+        The inner HTML of the element.
```

Comment:
```suggestion
        The inner HTML of the element as a string.
```

Reply:
I think the type of the return will be included so maybe not needed.